### ISSUE TITLE
Jaws is not reading out the string defined with aria-live="assertive" aria-atomic="true" aria-relevant="additions"

### ISSUE THREAD

**Issue by lalitapadhi9** [FreedomScientific/VFO-standards-support#54]
### Summary
> Example:
```

<!doctype html>
<html>
<head>
<meta charset="UTF-8">
<title>aria-atomic &amp; aria-relevant on aria-live regions</title>
<script src="http://code.jquery.com/jquery.js"></script>
<script>

function updateCharsLeft6() {
	var remaining = 280 - $('#message6').val().length;
	$('#chars6').text(remaining);
}
$(document).ready(function(e) {
	$('#message6').on('input propertychange', updateCharsLeft6);
});

</script>
</head>

<body>
<textarea name="message" id="message6" aria-describedby="input-info6" aria-required="true"></textarea>
    <br>
 <span id="LiveRegion1" aria-live="assertive" aria-atomic="true" aria-relevant="additions">The most populous metropolitan area in Florida is the Tampa/Saint Petersburg/Clearwater area. Characters remaining: <span id="chars6">280</span></span>
</body>
</html>


<span id="LiveRegion1"  aria-live="assertive" aria-atomic="true" aria-relevant="additions">The most populous metropolitan area in Florida is the Tampa/Saint Petersburg/Clearwater area.</span>

</body>
</html>

```

### Expected result

> What did you expect to happen?
>When opened with IE and jaws combination , the string defined in the span should have been read out

[>](url) What actually did happen?

>When opened with IE and jaws combination , the string defined in the span was not read out even though aria-live="assertive"

#### JAWS version and build number
Jaws 18(18.0.2946)
 
#### Operating System and version
Windows 10

#### Browser and version:

IE11

**Comment by stevefaulkner** [comment]
@lalitapadhi9 Are there browser/screen reader combinations where the content is announced? As simply declaring something as a live region without updating the content, or updating the display of the content (`display:none to display:block` for example), is unlikely to cause a screen reader to announce the content as a live region.

**Comment by lalitapadhi9** [comment]
yes Narrator and Edge combination reads it out

**Comment by stevefaulkner** [comment]
@lalitapadhi9 I have tested this test case with narrator and Edge https://s.codepen.io/stevef/debug/VQBQER
upon loading the page nothing was announced.

**Comment by lalitapadhi9** [comment]
Updating the example 

<!doctype html>
<html>
<head>
<meta charset="UTF-8">
<title>aria-atomic &amp; aria-relevant on aria-live regions</title>
<script src="http://code.jquery.com/jquery.js"></script>
<script>

function updateCharsLeft6() {
	var remaining = 280 - $('#message6').val().length;
	$('#chars6').text(remaining);
}
$(document).ready(function(e) {
	$('#message6').on('input propertychange', updateCharsLeft6);
});

</script>
</head>

<body>
<textarea name="message" id="message6" aria-describedby="input-info6" aria-required="true"></textarea>
    <br>
 <span id="LiveRegion1" aria-live="assertive" aria-atomic="true" aria-relevant="additions">The most populous metropolitan area in Florida is the Tampa/Saint Petersburg/Clearwater area. Characters remaining: <span id="chars6">280</span></span>
</body>
</html>

**Comment by stevefaulkner** [comment]
Hi @lalitapadhi9 what I need is an actual working example where the behavior you describe is occurring. Providing fragments of non working code is not helpful.

**Comment by lalitapadhi9** [comment]
Hi Steve,

The code snippet I have mentioned is a good example to check.

So basically in Edge when we add anything in the textarea, the narrator screen reader reads out the span id="LiveRegion1"

But the same output is not seen in jaws IE combination

Even though we have  aria-live="assertive" aria-atomic="true" aria-relevant="additions" defined in the span

**Comment by stevefaulkner** [comment]
@lalitapadhi9 I will need you to provide a working codepen or similar as I cannot get the code you provided to work

**Comment by rapomon** [comment]
You only need aria-live="assertive" and role="alert" attribute.

**Comment by bergatron** [comment]
Here's an example from the Freedom Scientific website: https://www.freedomscientific.com/Training/Surfs-Up/AriaLiveRegions.htm

When I open this page in Safari, VoiceOver announces the contents of the live region dynamically. When I open the same page in FF 63.0 + JAWS 2018.1808.10, the live region is not announced.

Perhaps there's a setting in JAWS I'm missing?

**Comment by gdubeIBM** [comment]
I am also receiving this issue. The aria-live attribute was being read properly on JAWS version 18 in FF but it is no longer working on the new 2018 version.

**Comment by stevefaulkner** [comment]
I just tested the test page cited https://www.freedomscientific.com/Training/Surfs-Up/AriaLiveRegions.htm using 63.0b8 (64-bit) and JAWS 2018.1809.1 and found the aria-live region being announced as expected. 

Things to **note**: as it's `aria-live=polite` it will not be announced until JAWS stops speaking other content. You can make this happen by pressing the `Ctrl` key. Also JAWS needs to be running before Firefox is opened.

**Comment by bergatron** [comment]
> Also JAWS needs to be running before Firefox is opened.

Ah thanks @stevefaulkner, I think that's the part I was missing. Just re-tested and the live region is now being announced as expected 👍

**Comment by gdubeIBM** [comment]
Having JAWS running before opening FF also resolved the issue on my end. Thanks for the speedy response 👍

**Comment by Mirodil** [comment]
It works on FF but not in MS Edge(Microsoft Edge 42.17134.1.0)

**Comment by ggordon-vfo** [comment]
From the comments it appears that this works in browsers other than Edge. Now that Microsoft is switching to Chromium as the Edge engine, we're not making additional fixes to Edge classic. Closing issue.

**Comment by IanLunn** [comment]
For anyone else coming here, Edge does not support JAWS: https://stackoverflow.com/questions/44166700/does-microsoft-edge-support-aria-live-regions

**Comment by SadhuSa** [comment]
Can jaws read out the text using aria-live="polite" in IE 11 ?  does IE11 Supports the Jaws 2019 screen reader for angular material applications. please suggest. Thanks!!

**Comment by chrisribe** [comment]
How can this issue be closed ?
Does the pull request [#1134](https://github.com/SAP/fundamental-react/pull/1134) fix this issue ??

I just lost so much time trying to figure out why aria-live=polite was not read under firefox with JAWS.... it's not funny,

**Comment by Mike-Diaz** [comment]
> How can this issue be closed ?
> Does the pull request [#1134](https://github.com/SAP/fundamental-react/pull/1134) fix this issue ??
> 
> Thanks @stevefaulkner for your comment !
> This solves it !?!? **"JAWS needs to be running before Firefox is opened."**
> I just lost so much time trying to figure out why aria-live=polite was not read under firefox with JAWS.... it's not funny,
> 
> @ggordon-vispero please confirm this was not closed by mistake, thank you.

Hey Chris, my PR was only for fixing something internally within our repo. Unfortunately I cannot answer as to why this issue is closed.
